build-system/app.js

```diff
@@ -936,6 +936,11 @@ app.get(['/dist/sw.js', '/dist/sw-kill.js', '/dist/ww.js'],
       next();
     });
 
+app.get('/dist/ampanalytics-lib.js', (req, res, next) => {
+  req.url = req.url.replace(/dist/, 'dist.3p/current');
+  next();
+});
```

Comment:
This actually belongs in the other PR. But given that a) it's just a rewrite rule and doesn't depend on the file actually existing, and b) we are going to merge 20->10442 then 10442->master, it's not hurting anything to have it here. But happy to move to other PR if requested.